clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -200,11 +200,11 @@ static unsigned getNumAttributeArgs(const ParsedAttr &AL) {
   return AL.getNumArgs() + AL.hasParsedType();
 }
 
-/// A helper function to provide Attribute Location for the Attr types
-/// AND the ParsedAttr.
-template <typename AttrInfo>
-static std::enable_if_t<std::is_base_of_v<Attr, AttrInfo>, SourceLocation>
-getAttrLoc(const AttrInfo &AL) {
+/// Helper functions to provide Attribute Location for the Attr types,
```

Comment:
These changes can be removed now right?

Reply:
My bad, I removed the overload but did not remember about that change, fixed now.